cmd/snap/cmd_snapshot.go

```diff
@@ -469,3 +488,31 @@ func (x *exportSnapshotCmd) Execute([]string) (err error) {
 
 	return nil
 }
+
+type importSnapshotCmd struct {
+	clientMixin
+	durationMixin
+	Positional struct {
+		Filename string `long:"filename"`
+	} `positional-args:"yes" required:"yes"`
+}
+
+func (x *importSnapshotCmd) Execute([]string) error {
+	filename := x.Positional.Filename
+	f, err := os.Open(filename)
+	if err != nil {
+		return fmt.Errorf("error accessing file: %v", err)
+	}
+	defer f.Close()
+
+	importSet, err := x.client.SnapshotImport(f)
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(Stdout, "Imported snapshot ID %d with data for snaps:\n", importSet.ID)
+	for _, snap := range importSet.Snaps {
+		fmt.Fprintf(Stdout, "- %s\n", snap)
+	}
```

Comment:
mmh maybe we should do what "snap save" does, and after "Imported snapshot..."  do the equivalent of "saved --id=ID" ?

Reply:
What happens if we don't save (e.g. someone SIGINTs at this moment?)

Reply:
If the import did not work the lines before with "x.client.SnapshotImport(f)" will return an error and this code is not reached (or am I misunderstanding the question?).